### DISCUSSION TITLE
Week 9 discussion

### DISCUSSION THREAD

**Discussion by kassyoveson** [USU-ITLS-x265-23-summer/class-repo#177] [General]
### Project Task List (Completed):

- wrote the basic html information needed for my form.

- Learned and downloaded visual studio code and postman

- Traveled to Switzerland & France, so I am taking it easy this week

### Next Tasks to be completed:

- Try bootstrap technique with colors and fonts for each section

- experiment with some text boxes into drop down lists

- Insert API either random food pictures or inspirational quotes of the day

- work on CSS elements

**Comment by derekscholes** [comment]
Have your created a Githunb page and set up a codespaces? I would love to see your code so that I can get ideas for my own project. If you have any questions or concerns let me know. Codespaces uses Visual Studio in the browser or you can set it up to use in the application which is cool. If you created any of these, please feel free to send me the link so that I can take a look.
